mmit/core/piecewise_function.cpp

```diff
@@ -146,7 +146,7 @@ bool has_slack_at_position(double func_breakpoint_pos, double func_sign, Coeffic
     }
 }
 
-int PiecewiseFunction::insert_point(double y, bool is_upper_bound) {
+int PiecewiseFunction::insert_point(double b, Coefficients F, bool is_upper_bound) {
```

Comment:
It's used at a few other places. I won't make the change now, but I'll try to think of a way to remove the dependency on this argument.